core/tests/puppeteer-acceptance-tests/utilities/user/logged-out-user.ts

```diff
@@ -951,6 +1007,198 @@ export class LoggedOutUser extends BaseUser {
+   */
+  async clickPartnerWithUsButtonInPartnershipsPage(): Promise<void> {
+    await this.clickButtonToNavigateToNewPage(
+      partnerWithUsButtonAtTheTopOfPartnershipsPage,
+      'Partner With Us button at the top of the Partnerships page',
+      partnershipsFormUrl,
+      'Partnerships Google Form'
+    );
+  }
+
+  /**
+   * Function to change the site language to the given language code.
+   * @param langCode - The language code to change the site language to. Example: 'pt-br', 'en'
+   */
+  async _changeSiteLanguage(langCode: string): Promise<void> {
+    const languageDropdown = '.e2e-test-language-dropdown';
+    const languageOption = `.e2e-test-i18n-language-${langCode} a`;
+    await this.clickOn(languageDropdown);
+    await this.clickOn(languageOption);
+    await this.page.waitForTimeout(2000);
+  }
+
+  /**
+   * Function to click the Partner With Us button in the Partnerships page
+   * and check if it opens the Partnerships Google form in Portuguese.
+   * The button is in the bottom section of the page.
+   */
+  async clickPartnerWithUsButtonInPartnershipsPageInGivenLanguage(
+    langCode: string
+  ): Promise<void> {
+    await this._changeSiteLanguage(langCode);
+
+    await this.clickButtonToNavigateToNewPage(
+      partnerWithUsButtonAtTheBottomOfPartnershipsPage,
+      'Partner With Us button at the bottom of the Partnerships page',
+      partnershipsFormInPortugueseUrl,
+      'Partnerships Google Form'
+    );
+  }
+
+  /**
+   * Function to click the Download Brochure button in the Partnerships page
+   * and check if it opens the Partnerships Brochure.
+   */
+  async clickDownloadBrochureButtonInPartnershipsPage(): Promise<void> {
+    await this.openExternalPdfLink(
+      brochureButtonInPartnershipsPage,
+      partnershipsBrochureUrl
+    );
+  }
+
+  /**
+   * Function to click the first Read blog post  in the Partnerships page
+   * and check if it opens the blog page.
+   */
+  async clickReadBlogPostLinkInPartnershipsPage(): Promise<void> {
+    await this.page.waitForXPath('//a[contains(text(),"Read blog post")]');
+    await this.clickButtonToNavigateToNewPage(
+      readBlogPostButtonInPartnershipsPage,
+      'Read blog post button',
+      blogPostUrlinPartnershipsPage,
+      'Blog Post'
+    );
+  }
+
+  /**
+   * Function to click the Read more stories button in the Partnerships page
+   * and check if it opens the blog page.
+   */
+  async clickReadMoreStoriesButtonInPartnershipsPage(): Promise<void> {
+    await this.clickButtonToNavigateToNewPage(
+      readMoreStoriesButtonInPartnershipsPage,
+      'Read more stories button',
+      blogUrl,
+      'Blog'
+    );
+  }
+
+  /**
+   * Function to click the Apply To Volunteer at the top of the Volunteer page
+   * and check if it opens the Volunteer form.
+   */
+  async clickApplyToVolunteerAtTheTopOfVolunteerPage(): Promise<void> {
+    await this.page.waitForSelector(
+      applyToVolunteerButtonAtTheTopOfVolunteerPage
+    );
```

Comment:
You won't need to wait for button before clicking on it.. If you use clickOn(). It takes care or that.